pkg/plugin/acceptance_testing.go

```diff
@@ -97,12 +97,12 @@ func testSpecifier_Specify_Success(t *testing.T, tdf testDispenserFunc) {
 		Version:     "v1.2.3",
 		Author:      "Donald Duck",
 		SourceParams: map[string]Parameter{
-			"param1.1": {Default: "foo", Required: false, Description: "Param 1.1 description"},
-			"param2.1": {Default: "bar", Required: true, Description: "Param 1.2 description"},
+			"param1.1": {Default: "foo", Description: "Param 1.1 description"},
+			"param2.1": {Default: "bar", Description: "Param 1.2 description", Validations: []Validation{{VType: ValidationTypeRequired}}},
```

Comment:
We need to set the `Type` in the expectation, then tests should pass (well, and we need to address [this comment](https://github.com/ConduitIO/conduit/pull/311#discussion_r832454638), to make sure `Validations` are actually `nil` if there are none).
```suggestion
			"param1.1": {Default: "foo", Type: "string", Description: "Param 1.1 description"},
			"param2.1": {Default: "bar", Type: "string", Description: "Param 1.2 description", Validations: []Validation{{VType: ValidationTypeRequired}}},
```